framework_tvb/tvb/interfaces/web/templates/jinja2/form_fields/datatype_select_field.html

```diff
@@ -1,13 +1,30 @@
-<p class="field-data">
-<select name="{{ field.name }}" id="{{ field.name }}" class="dataset-selector" tabindex="1">
+<script type="text/javascript" src="/static/js/filters.js?{{ currentVersionJS }}"></script>
+
+<div class="field-data">
+<select name="{{ field.name }}" id="{{ field.name }}" class="dataset-selector" tabindex="1"
+        onchange="updateDivContent('data_time_series', this, ''); updateDatatypeDiv(this)" title="Fully Loaded!">
 {% for option in field.options() %}
     <option value="{{ option.value }}" {{ 'selected' if option.checked }} class="form-control">{{ option.label }}</option>
 {% endfor %}
 </select>
 {% if field.draw_dynamic_conditions_buttons %}
-    <p class="field-adapters" id="{{ field.owner.name }}{{ field.name }}data_select">
-        <input type="button" tabindex='1' value="Add Filter" onclick="addFilter('{{ field.owner.name }}' + '{{ field.name }}' + 'data_select', {{ field.get_dynamic_filters }})"/>
-        <input type="button" tabindex='1' value="Apply Filters" onclick="refreshData('{{ field.owner.name }}', '{{ field.name }}' + 'data_select', '{{ field.name }}', 'defaultTree')"/>
-    </p>
+    <div class="field-adapters" id="{{ field.owner.name }}{{ field.name }}data_select">
```

Comment:
Here again, I would prefer if you do not change the tag from `<p>` into `<div>` unless there is something really not working, otherwise the current CSS schema might break

Reply:
Done.

Reply:
You say `done` but still you changed from `<p>` into `<div>` and did not give a justification why this change is necessary.

Reply:
Oh yes, I changed two <p>'s actually. The first one (the one from above in the code) can stay <p> but the second one not really... Because it won't include the <div>'s for the hidden form filters inside the second p, but outside them. I checked this on stackoverflow and in the html documentation it says that these kind of things can happen because inline elements are not supposed to contain block elements.

If we really wan't to keep it <p> then I need to find a different way to include the hidden filters new to the user defined ones.
![image](https://user-images.githubusercontent.com/32891492/87917173-3826a180-ca7d-11ea-8096-adee2a747ca6.png)